gulp/js/default.js

```diff
@@ -57,32 +59,44 @@ var taskName = 'js',
 			// Create a map of entries, i.e. {'assets/js/main': './source/assets/js/main.js'}
 			entry: helpers.webpack.getEntries(src, config.srcBase),
 			module: {
-				loaders: [
+				rules: [
 					{
 						test: /\.hbs$/,
-						loader: 'handlebars-loader'
+						loader: 'handlebars-loader',
+						options: {
+							partialResolver: function(partial, callback) {
+								var resolvedPath = path.resolve('./source/', partial + '.hbs');
+
+								callback(null, resolvedPath);
+							}
+						}
 					},
 					{
 						test: /jquery\.js$/,
-						loader: 'expose?$!expose?jQuery'
+						loader: 'expose-loader?$!expose-loader?jQuery'
 					},
 					{
 						test: /handlebars\.js$/,
-						loader: 'expose?Handlebars'
+						loader: 'expose-loader?Handlebars'
 					},
 					{
 						test: /(\.js|\.jsx)$/,
 						exclude: /node_modules/,
 						loader: 'babel-loader',
-						query: {
-							presets: ['es2015', 'react'],
-							plugins: [
-								// Work around some issues in IE
-								'transform-class-properties',
-								'transform-proto-to-assign',
-								['transform-es2015-classes', {
+						options: {
+							presets: [
+								['env', {
+									debug: util.env.dev,
+									useBuiltIns: 'entry',
```

Comment:
Let's rather check`usage`: https://github.com/babel/babel/tree/master/experimental/babel-preset-env#usebuiltins-usage

Reply:
Note: Dependencies need to be updated accordingly:
```
    "@babel/core": "^7.0.0-beta.34",
    "@babel/polyfill": "^7.0.0-beta.34",
    "@babel/preset-env": "^7.0.0-beta.34",
    "babel-loader": "^8.0.0-beta.0",
```